packages/client/modules/userDashboard/components/GcalModal/GcalModal.tsx

```diff
@@ -0,0 +1,173 @@
+import styled from '@emotion/styled'
+import dayjs from 'dayjs'
+import React, {useState} from 'react'
+import DialogContainer from '../../../../components/DialogContainer'
+import DialogContent from '../../../../components/DialogContent'
+import DialogTitle from '../../../../components/DialogTitle'
+import PrimaryButton from '../../../../components/PrimaryButton'
+import {PALETTE} from '../../../../styles/paletteV3'
+import DateTimePicker from './DateTimePicker'
+import Checkbox from '../../../../components/Checkbox'
+import StyledError from '../../../../components/StyledError'
+import PlainButton from '../../../../components/PlainButton/PlainButton'
+import {Close} from '@mui/icons-material'
+import {CreateGcalEventInput} from '../../../../__generated__/CreateGcalEventMutation.graphql'
+import useForm from '../../../../hooks/useForm'
+import Legitity from '../../../../validation/Legitity'
+
+const Wrapper = styled('div')({
+  display: 'flex',
+  justifyContent: 'flex-end',
+  paddingTop: 16
+})
+
+const StyledInput = styled('input')({
```

Comment:
There are quite a few styled components in this PR. I built most of it during Hack Week and copied existing components as it'd be quicker. I can refactor some of these if it's a problem